src/eventDispatcher.js

```diff
@@ -18,7 +18,10 @@ class EventDispatcher {
      * @param {Function} handler. 
      */
     bind(eventName, handler) {
-        if (typeof handler === 'function') {
+
+        if (typeof eventName !== 'string' || typeof handler !== 'function') {
```

Comment:
When you check parameters and you throw an error there is no need for IF / Else, since once you throw an error the execution is broken. Having that in mind, please do the following:
```
if (typeof ...) { 
 throw...
}

if (this. eventDispatcherMap) {
 ...
}
```

Reply:
Additional note, throwing of an error should be a precise operation. Message _First parameter is not string or second parameter is not function_ should be improved, since it is not clear where the error is. The conclusion is to check each parameter independently. On top of that, you should not use _first parameter, second parameter_, since caller then needs to count parameters in order to see which one is wrong. 
```
if (typeof eventName !== 'string') {
 throw new SyntaxError("eventName should be a string);
}
```